### DISCUSSION TITLE
Implementation of Uav-core on real drone

### DISCUSSION THREAD

**Discussion by throwawayacco** [ctu-mrs/mrs_uav_system#57] [Q&A]
I'm currently working with a pixhawk based drone. When simulating with your system (mrs-uav system with mpc-controller / tracker) I noticed that it gave better results than our current system and simulations with the drone. I was hoping to implement your system on our current drone, but I can't see a guide on how to do it on your github page. 

I see the page about the px4 configuration but it seems to be only the pixhawk configuration you use and not related to the uav-system you've made.

So is there anywhere I can look for a guide on how to implement the simulated system on a real drone?

**Comment by spurnvoj** [comment]
Hi, as you mentioned, page https://ctu-mrs.github.io/docs/hardware/px4_configuration.html explains how to set up hardware, but there is no tutorial about how to set up the software. However, we should probably create a page to mention these steps too.

In general: 

- When we fly with real drones, we change some of the .bashrc variables to correspond to hw setup (and change the RUN_TYPE to uav). 
- We also use tmux scripts instead of tmuxinator - https://github.com/ctu-mrs/uav_core/tree/master/tmux_scripts
